test/e2e/network/ingress.go

```diff
@@ -935,3 +939,176 @@ func detectNegAnnotation(f *framework.Framework, jig *e2eingress.TestJig, gceCon
 		framework.ExpectNoError(err)
 	}
 }
+
+var _ = SIGDescribe("Ingress API", func() {
+	f := framework.NewDefaultFramework("ingress")
+	/*
+		Release: v1.19
+		Testname: Ingress API
+		Description:
+		- The networking.k8s.io API group MUST exist in the /apis discovery document.
+		- The networking.k8s.io/v1beta1 API group/version MUST exist in the /apis/networking.k8s.io discovery document.
+		- The ingresses resources MUST exist in the /apis/networking.k8s.io/v1beta1 discovery document.
+		- The networking/ingresses resource must support create, get, list, watch, update, patch, delete, and deletecollection.
+	*/
+
+	ginkgo.It("should support creating Ingress API operations", func() {
+		// Setup
+		ns := f.Namespace.Name
+		ingVersion := "v1beta1"
+		ingClient := f.ClientSet.NetworkingV1beta1().Ingresses(ns)
```

Comment:
since this is a namespaced resource, we also want to exercise cluster-wide list and watch... basically duplicate the list/watch case and do them with a `f.ClientSet.NetworkingV1beta1().Ingresses("")` client

Reply:
I added the cluster-wide tests. Let me know if these are lacking.